front/app/src/main.tsx

```diff
@@ -17,5 +16,17 @@ ReactDOM.createRoot(document.getElementById("root") as HTMLElement).render(
 					<App />
 			</PersistGate>
 		</Provider>
-	</React.StrictMode>,
 );
+
+// ReactDOM.createRoot(document.getElementById("root") as HTMLElement).render(
+// 	<React.StrictMode>
+// 		<Provider store={store}>
+// 			<PersistGate
+// 				loading={<p>recover data from local storage...</p>}
+// 				persistor={persistor}
+// 				>
+// 					<App />
+// 			</PersistGate>
+// 		</Provider>,
+// 	</React.StrictMode>,
+// );
```

Comment:
Commented part up here

Reply:
I commented it, because strictMode is used on dev mode of react but it interfer with the backtrack debugger.